### DISCUSSION TITLE
What does the icons mean?

### DISCUSSION THREAD

**Discussion by mjoint** [iterate-ch/cyberduck#14617] [Q&A] [answered]
Hi. I can't find a description what the different icons mean.
Especially that red cross.

Have a nice day,
John 

<img width="352" alt="PastedGraphic-1" src="https://user-images.githubusercontent.com/132555135/236149509-c7efeaa5-b2aa-48be-9fd5-dbdff4e456b5.png">

**Comment by dkocher** [comment, marked as answer]
This is documented in [Status of Files](https://docs.mountainduck.io/mountainduck/sync/#status-of-files).

**Comment by mjoint** [comment]
Perfect, thanks! Feels like I should been able to find that my self.. 😅
/John
